src/js/components/account/awards/LegacyResultsTable.jsx

```diff
@@ -8,16 +8,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import moment from 'moment';
-import { formatMoney } from 'helpers/moneyFormatter';
-
 import IBTable from 'components/sharedComponents/IBTable/IBTable';
 
 import ResultsTableGenericCell from 'components/search/table/cells/ResultsTableGenericCell';
 import ResultsTableAwardIdCell from 'components/search/table/cells/ResultsTableAwardIdCell';
 
 import LegacyTableHeaderCell from './LegacyTableHeaderCell';
 
+import AccountTableSearchFields from '../../../dataMapping/search/accountTableSearchFields';
```

Comment:
o rly? You should be able to use the module path from `src/js/`

Reply:
Fixed